tests/devices/fetch_network.js

```diff
@@ -2,17 +2,20 @@
 
 import assert from "assert/strict";
 import url from "node:url";
+import { Worker } from "node:worker_threads";
 
 const __dirname = url.fileURLToPath(new URL(".", import.meta.url));
 process.on("unhandledRejection", exn => { throw exn; });
 
 const USE_VIRTIO = !!process.env.USE_VIRTIO;
-
 const TEST_RELEASE_BUILD = +process.env.TEST_RELEASE_BUILD;
-const { V86 } = await import(TEST_RELEASE_BUILD ? "../../build/libv86.mjs" : "../../src/main.js");
 
+const { V86 } = await import(TEST_RELEASE_BUILD ? "../../build/libv86.mjs" : "../../src/main.js");
 const SHOW_LOGS = false;
 
+var SERVER_PORT = parseInt(process.env.SERVER_PORT, 10) || 0;
```

Comment:
Is there still a need to specify this using an env variable? If not, I'd suggest removing it and simplifying the remaining code (that is, the worker always sends a random port to the caller).

Reply:
It's for debugging. When I wrote that server worker, I wanted to make similar test for browser (like a [manual/gc.html](https://github.com/copy/v86/blob/master/tests/manual/gc.html)) but abandoned this idea later.